commit 26f9d62b9a9b4bfa736b342903d16dc0d8739d98
Author: Allen Winter <winter@kde.org>
Date:   Thu May 31 12:58:07 2012 -0400

    Revert last: Remove X11 requirement
    
    CCMAIL: djarvie@kde.org
    David, without explicit linking to X11 libs, I get an unresolved
    symbol for XSetWMHints

diff --git a/kalarm/CMakeLists.txt b/kalarm/CMakeLists.txt
index 9efb9a1..97e69a8 100644
--- a/kalarm/CMakeLists.txt
+++ b/kalarm/CMakeLists.txt
@@ -202,6 +202,7 @@ if(KALARM_USE_AKONADI)
         kdepim
         ${KDE4_KCMUTILS_LIBS}
         ${KDE4_PHONON_LIBS}
+        ${X11_X11_LIB}
     )
 else(KALARM_USE_AKONADI)
     target_link_libraries(kalarm_bin
@@ -220,6 +221,7 @@ else(KALARM_USE_AKONADI)
         kdepim
         ${KDE4_KCMUTILS_LIBS}
         ${KDE4_PHONON_LIBS}
+        ${X11_X11_LIB}
     )
 endif(KALARM_USE_AKONADI)
 
